tests/functional/DatadirTest.php

```diff
@@ -46,6 +49,41 @@ protected function setUp(): void
         }
     }
 
+    /**
+     * @dataProvider provideDatadirSpecifications
+     */
+    public function testDatadir(DatadirTestSpecificationInterface $specification): void
+    {
+        $tempDatadir = $this->getTempDatadir($specification);
+
+        $process = $this->runScript($tempDatadir->getTmpFolder());
+
+        if ($process->getOutput()) {
+            $this->output = json_decode($process->getOutput(), true);
+        }
+
+        // Load tearDown.php file
```

Comment:
:+1: 

Asi by sme to uz  mohli pridat aj do `datadir` lib, celkom sa to ujalo.